coverage.txt

```diff
@@ -1,17 +1,18 @@
 mode: atomic
-github.com/voi-oss/svc/config.go:10.44,11.42 1 0
-github.com/voi-oss/svc/config.go:14.2,14.55 1 0
-github.com/voi-oss/svc/config.go:17.2,17.12 1 0
-github.com/voi-oss/svc/config.go:11.42,13.3 1 0
-github.com/voi-oss/svc/config.go:14.55,16.3 1 0
-github.com/voi-oss/svc/http.go:21.87,31.2 2 0
-github.com/voi-oss/svc/http.go:34.53,38.2 2 0
-github.com/voi-oss/svc/http.go:41.38,43.2 1 0
-github.com/voi-oss/svc/http.go:46.34,48.71 2 0
-github.com/voi-oss/svc/http.go:51.2,51.12 1 0
-github.com/voi-oss/svc/http.go:48.71,50.3 1 0
-github.com/voi-oss/svc/http.go:55.40,57.2 1 0
-github.com/voi-oss/svc/logger.go:13.102,28.2 5 19
+github.com/voi-oss/svc/config.go:11.44,13.2 1 2
```

Comment:
Why is this file committed? :o

Reply:
It was modified when running `make test` locally.